osu.Game/Screens/Play/Player.cs

```diff
@@ -323,23 +325,38 @@ private void initializeStoryboard(bool asyncLoad)
 
         protected FailOverlay FailOverlay { get; private set; }
 
+        private void fail()
+        {
+            GameplayClockContainer.Stop();
+            HasFailed = true;
+            FailOverlay.Retries = RestartCount;
+            FailOverlay.Show();
+        }
         private bool onFail()
         {
+            //issue #3372
+            if (Beatmap.Value.Mods.Value.Any(x => x is ModEasy))
```

Comment:
``Player.cs`` shouldn't be modified to support single mods. Instead all of this should be done inside ``ModEasy``.

Reply:
oh thanks. I'll move it right now.